tests/ct_connection_vhost_ports_with_netperf_udp.py

```diff
@@ -0,0 +1,176 @@
+
+        # create VMs
+        result, vm_name = test_utils.vm_create(self.config_data['vm_location_list'])
+        if not result:
+            self.result.addFailure(self, sys.exc_info())
+            self.fail(f"VM creation failed for {vm_name}")
+
+
+        # create telnet instance for VMs created
+        self.conn_obj_list = []
+        vm_cmd_list = []
+        vm_id = 0
+        for vm, port in zip(self.config_data['vm'], self.config_data['port']):
+           globals()["conn"+str(vm_id+1)] = connectionManager("127.0.0.1", f"655{vm_id}", vm['vm_username'], vm['vm_password'], timeout=20)
+           self.conn_obj_list.append(globals()["conn"+str(vm_id+1)])
+           globals()["vm"+str(vm_id+1)+"_command_list"] = [f"ip addr add {port['ip_address']} dev {port['interface']}", f"ip link set dev {port['interface']} up", f"ip link set dev {port['interface']} address {port['mac_local']}", f"ip route add {port['ip_route']} via {port['ip_add_route']} dev {port['interface']}",f"ip neigh add dev {port['interface']}  {vm['remote_ip']} lladdr {vm['mac_remote']}"]
+           vm_cmd_list.append(globals()["vm"+str(vm_id+1)+"_command_list"])
+           vm_id+=1
+          
+
+        # configuring VMs
+        for i in range(len(self.conn_obj_list)):
+            log.info(f"Configuring VM{i}....")
+            test_utils.configure_vm(self.conn_obj_list[i], vm_cmd_list[i])
+            log.info(f"execute ethtool {self.config_data['port'][i]['interface']} offload on VM{i}")
+            if not test_utils.vm_ethtool_offload(self.conn_obj_list[i],self.config_data['port'][i]['interface'] ):
+                  self.result.addFailure(self, sys.exc_info())
+                  self.fail(f"FAIL: failed to set ethtool offload {self.config_data['port'][i]['interface']} on VM{i}")
+
+        # Netperf and netserver for local host
+        time.sleep(30)
+        for i in range(len(self.conn_obj_list)):
+            if not test_utils.vm_check_netperf(self.conn_obj_list[i], f"VM{i}"):
+                self.result.addFailure(self, sys.exc_info())
+                self.fail(f"FAIL: netperf is not install on VM{i}")
+        i=0
+        log.info(f"Start netserver on VM{i}")
+        if not test_utils.vm_start_netserver(self.conn_obj_list[i]):
+              self.result.addFailure(self, sys.exc_info())
+              self.fail(f"FAIL: failed to start netserver on VM{i}")
+        log.info(f"netserver started on VM{i}")      
+        
+        #send netperf from local VM
+        i=1
+        log.info(f"Initially execute netperf on VM{i}")
+        if not test_utils.vm_netperf_client(self.conn_obj_list[i], self.config_data['vm'][i]['remote_ip'],
+                self.config_data['netperf']['testlen'], self.config_data['netperf']['testname'], option = self.config_data['netperf']['cmd_option']):
+            self.result.addFailure(self, sys.exc_info())
+            self.fail(f"FAIL: failed to start netperf")
+
+        self.dataplane.kill()
```

Comment:
With previous comment, as no scapy involved, dataplane related code is removed. This line is needed to remove too.